e3db/types/client_details.py

```diff
@@ -103,6 +105,23 @@ def public_key(self):
         else:
             return self.__public_key['curve25519']
 
+    # public_signing_key getters
+    @property
+    def public_signing_key(self):
+        """
+        Get public_signing_key of Client.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        str
+            Base64 URLencoded public_signing_key of Client.
+        """
+        return self.__public_signing_key['ed25519']
```

Comment:
do we need to switch based off the crypto mode (e.g. nist) what signing key is returned?

Reply:
I don't believe so, though I'm only cross-referencing with other SDKs. In other implementations of `ClientDetails` only `ed25519` is referenced